### ISSUE TITLE
Old News 

### ISSUE THREAD

**Issue by sajedazoabi** [facn2/knafeh#53]
Your API's should bring the last updated News, not old ones, there are many old News and as a visitor to Israel I want to  be able to read the latest Israeli news

**Comment by tyeung2017** [comment]
Yeah we think the same thing, apparently guardian API does not refresh the news for Israel in a timely manner 

If you know any good israel news source with API, please let us know.
